helusers/jwt.py

```diff
@@ -60,6 +62,29 @@ def validate(self, keys, audience, required_claims=_NOT_PROVIDED):
             if len(set(audience).intersection(claim_audiences)) == 0:
                 raise ValidationError("Invalid audience.")
 
+    def validate_issuer(self):
+        try:
+            issuer = self.issuer
+        except KeyError:
+            raise ValidationError('Required "iss" claim is missing.')
+
+        if issuer not in self.settings.ISSUER:
```

Comment:
`self.settings.ISSUER` can be a string, right? Should it be turned into a list first (or some other iterable)?

There's the `_Defaults` class in `oidc` module (within the `_build_defaults` function) that does that kind of thing. But it doesn't support custom user supplied settings, only regular Django settings. Could that be changed to be usable here too somehow?

Reply:
I added same kind of string to list as in the _build_defaults for now.

The settings situation is a bit hairy. I think we could see if we could remove the two different Settings classes altogether at some point.